src/src/com/microsoft/aad/adal/AuthenticationContext.java

```diff
@@ -1707,6 +1701,11 @@ private TokenCacheItem findFamilyItemForUser(final UserIdentifierType userIdenti
         Logger.v(TAG + methodName, "No family items found in the cache.");
         return null;
     }
+
+    private boolean isRefreshTokenForFamily(TokenCacheItem tokenCacheItem) {
+        return (!StringExtensions.IsNullOrBlank(tokenCacheItem.getFamilyClientId())
```

Comment:
I don't think that we actually want to know for each token why we don't consider it for usage (whether it's not a family token or it's not for our user). It's usually enough to know that we couldn't find anything useful for us